### ISSUE TITLE
[Bug] MAC address for Ethernet changes after each reboot

### ISSUE THREAD

**Issue by blakadder** [darkxst/ha-operating-system#1]
MAC address on the Ethernet interface changes after each reboot. 

Possible fix https://forum.armbian.com/topic/28005-mac-adress-change-everyboot-orange-5

**Comment by darkxst** [comment]
As a workaround setting a cloned mac address via network manager nmcli should work.
`nmcli con modify <uuid> ethernet.cloned-mac-address <mac addr>`


However I took a brief look at this arvo, things are vastly different between the Rockchip kernel vs mainline kernel (which we have here). From what I saw, It seems for most of the Rockchip SoC's, U-Boot will read the cpuid from OTP and derive a constant mac address from that which gets inserted into the device tree for the kernel to grab.  OTP is not hooked up yet, and rv1126 may need soc specific patches to u-boot for it work.
